### PR TITLE
Fix TravisCI

### LINKED ISSUE #209: Fix Travis builds
```
 ---> Running in 2510f340c950
[NOTE] travis-opam is not installed.
Nothing to do.
 ---> ea0249222a44
Removing intermediate container 2510f340c950
Step 6/13 : RUN opam switch 4.05.0
 ---> Running in aa291b251d25
[ERROR] No switch 4.05.0 is currently installed. Did you mean 'opam switch create 4.05.0'?
        Installed switches are:
          - 4.07
          - 4.06
          - 4.05
          - 4.04
          - 4.03
The command '/bin/sh -c opam switch 4.05.0' returned a non-zero code: 5
The command "bash -ex ./.travis-docker.sh" exited with 5.
```

### PR DESCRIPTION
**Pull request by samoht**
Fix #209

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -11,10 +11,10 @@ env:
...
-  - DISTRO="ubuntu-16.04" OCAML_VERSION="4.05.0"
-  - DISTRO="alpine" OCAML_VERSION="4.06.0"
-  - DISTRO="fedora-25" OCAML_VERSION="4.03.0"
+  - DISTRO="debian-stable" OCAML_VERSION="4.05"
+  - DISTRO="debian-testing" OCAML_VERSION="4.03"
+  - DISTRO="debian-unstable" OCAML_VERSION="4.04"
+  - DISTRO="debian-unstable" OCAML_VERSION="4.06"
+  - DISTRO="ubuntu-16.04" OCAML_VERSION="4.05"
+  - DISTRO="alpine" OCAML_VERSION="4.06"
+  - DISTRO="fedora-25" OCAML_VERSION="4.03"
```
Comment (XVilka) [inline .travis.yml]: Seems also require fresher version, like "fedora-28"

Comment (samoht) [reply .travis.yml]: thx, updated

**Comment by XVilka** [comment]
Seems problem with some tests on MSVC/Windows:
```
[ERROR]             alignment          3   aligned to large.
- -- bounds.035 [test_copyv_overflow.] Failed --
- in _build/_tests\bounds.035.output:
- [exception] Out of memory
- 
- The full test results are available in `_build/_tests`.
- 2 errors! in 2.108s. 63 tests run.
[ERROR] The compilation of cstruct failed at "jbuilder runtest -p cstruct".
#=== ERROR while compiling cstruct.3.2.1 ======================================#
# opam-version         1.3.0~dev (9fc27413c08030d909802dcdfb6e63b75d9dd762)
# os                   win32
# command              jbuilder runtest -p cstruct
# path                 C:/cygwin64/home/appveyor/.opam/4.05.0+msvc32c/build/cstruct.3.2.1
# exit-code            1
# env-file             C:/cygwin64/home/appveyor/.opam/4.05.0+msvc32c/build/cstruct.3.2.1\cstruct-308-58c514.env
# stdout-file          C:/cygwin64/home/appveyor/.opam/4.05.0+msvc32c/build/cstruct.3.2.1\cstruct-308-58c514.out
# stderr-file          C:/cygwin64/home/appveyor/.opam/4.05.0+msvc32c/build/cstruct.3.2.1\cstruct-308-58c514.err
### stderr ###
# Testing cstruct.
# [...]
[OK]                append             0   append is concat.
[OK]                alignment          0   aligned to 4096.
[OK]                alignment          1   aligned to 512.
[OK]                alignment          2   aligned to 0.
[ERROR]             alignment          3   aligned to large.
# -- bounds.035 [test_copyv_overflow.] Failed --
# in _build/_tests\bounds.035.output:
# [exception] Out of memory
# 
# The full test results are available in `_build/_tests`.
# 2 errors! in 2.108s. 63 tests run.
```

**Comment by samoht** [comment]
Interestingly, it was failing on win32 before as well:

```
==============================================================================
- Error: misc tests:4:alignment:3:aligned to large
- 
- (Invalid_argument "check_alignment must be positive integer")
- ------------------------------------------------------------------------------
- Ran: 11 tests in: 0.09 seconds.
- FAILED: Cases: 11 Tried: 11 Errors: 1 Failures: 0 Skip:  0 Todo: 0 Timeouts: 0.
-       bounds alias lib_test/runtest
- ....................................................
```
but for some reasons, the CI was happy. I'll investigate..

**Comment by samoht** [comment]
All green at least!

**Pull request merged**